clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -28,37 +28,37 @@
  * This is applicable when the consumer is having Kafka auto-manage group membership. If the consumer directly assigns partitions,
  * those partitions will never be reassigned and this callback is not applicable.
  * <p>
- * When Kafka is managing the group membership, a partition re-assignment will be triggered any time the members of the group change or the subscription
+ * When Kafka is managing the group membership, a partition re-assignment will be triggered whenever the members of the group change or the subscription
  * of the members changes. This can occur when processes die, new process instances are added or old instances come back to life after failure.
- * Partition re-assignments can also be triggered by changes affecting the subscribed topics (e.g. when the number of partitions is
+ * Partition re-assignments can also be triggered by changing the subscribed topics (e.g. when the number of partitions is
```

Comment:
I guess it's about interpretation, but I actually find the original version much clearer. "changing the subscribed topics" seems a bit confusing, as its what we would use to refer to "changing the subscribed topic from T1 to T2", but here we want to refer to changes affecting the topics (ex. partition count changes)

Reply:
Got it. Thanks for your explanation.